modules/hcp-migrate-aws-single-to-multiarch.adoc

```diff
@@ -0,0 +1,168 @@
+----
+
+.. Save the multi-architecture release image name in the `IMAGE` variable by running the following command:
++
+[source,terminal]
+----
+IMAGE=quay.io/openshift-release-dev/ocp-release:$MULTI_ARCH_TAG
+----
+
+.. Get the list of the multi-architecture image digests, by running the following command:
++
+[source,terminal]
+----
+$ oc image info $IMAGE
+----
++
+.Example output
+[source,terminal]
+----
+OS            DIGEST
+linux/amd64   sha256:b4c7a91802c09a5a748fe19ddd99a8ffab52d8a31db3a081a956a87f22a22ff8
+linux/ppc64le sha256:66fda2ff6bd7704f1ba72be8bfe3e399c323de92262f594f8e482d110ec37388
+linux/s390x   sha256:b1c1072dc639aaa2b50ec99b530012e3ceac19ddc28adcbcdc9643f2dfd14f34
+linux/arm64   sha256:7b046404572ac96202d82b6cb029b421dddd40e88c73bbf35f602ffc13017f21
+----
+
+. Transition the hosted cluster from single-architecture to multi-architecture:
+
+.. Set the `multiArch` flag to `true` on your hosted cluster so that the hosted cluster can create and manage both 64-bit AMD and 64-bit ARM node pools. Run the following command:
++
+[source,terminal]
+----
+$ oc patch -n clusters hostedclusters/<hosted_cluster_name> -p '{"spec":{"platform":{"aws":{"multiArch":true}}}}' --type=merge
+----
+
+.. Set the multi-architecture {product-title} release image for the hosted cluster. Ensure that you use the same {product-title} version as the hosted cluster. Run the following command:
++
+[source,terminal]
+----
+$ oc patch -n clusters hostedclusters/<hosted_cluster_name> -p '{"spec":{"release":{"image":"quay.io/openshift-release-dev/ocp-release:<4.x.y>-multi"}}}' --type=merge <1>
+----
+<1> Replace `<4.y.z>` with the supported {product-title} version that you use.
+
+.. Confirm that the multi-architecture image is set in the hosted cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get hostedcluster/<hosted_cluster_name> -o jsonpath='{.spec.release.image}'
+----
+
+. Check that the status of the `HostedControlPlane` resource is `Progressing` by running the following command:
++
+[source,terminal]
+----
+$ oc get hostedcontrolplane -n <hosted_control_plane_namespace> -oyaml
+----
++
+.Example output
+[source,yaml]
+----
+...
+  - lastTransitionTime: "2024-07-28T13:07:18Z"
+    message: HostedCluster is deploying, upgrading, or reconfiguring
+    observedGeneration: 5
+    reason: Progressing
+    status: "True"
+    type: Progressing
+...
+----
+
+. Check that the status of the `HostedCluster` resource is `Progressing` by running the following command:
++
+[source,terminal]
+----
+$ oc get hostedcluster <hosted_cluster_name> -n <hosted_cluster_namespace> -oyaml
+----
+
+.Verification
+
+* Verify that the multi-architecture {product-title} release image is updated in your `HostedControlPlane` resource by running the following command:
```

Comment:
Should this be something like Verify the node pool is using the multi-architecture release image by entering the following command? It kind of reads like a multi-arch release image was updated from some previous version.